falcon/request.py

```diff
@@ -524,6 +534,12 @@ def cookies(self):
         return self._cookies.copy()
 
     @property
+    def json(self):
```

Comment:
Also this is so trivial that there is no point in splitting implementation into two methods.